docs/source/troubleshooting/changelog.rst

```diff
@@ -523,18 +523,24 @@ New format
                 chunks_to: "natural"
         RUNNING: chunk
 
-.. figure:: fig/new_dependencies_0.png
-   :name: new_dependencies_0
-   :align: center
-   :alt: new_dependencies
+
+.. autosubmitfigure::
+    :command: create
+    :expid: a000
+    :type: png
+    :figure: new_dependencies.png
+    :name: new_dependencies
+    :width: 100%
+    :align: center
+    :alt: new_dependencies
```

Comment:
Note: path is not needd, and the `figure` has only the file name... we are moving to somewhere like `build/html/_static/troubleshooting/new_dependencies.png`... so images must be unique per file, but if that becomes a problem, later we can include the `rst` file name in the target path.